Makefile

```diff
@@ -69,6 +69,13 @@ go_lint: golint go_test_tag_lint
 	golint -set_exit_status ./webapp/...
 	golint -set_exit_status ./webdriver/...
 
```

Comment:
The installation for `golangci-lint` uses the recommended binary installation instead of a `go install`. Let me know if this looks good to you!

Also, I assume that the Dockerfile change is no longer needed with this addition?